sphinx/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line, and also
+# from the environment for the first two.
+SPHINXOPTS    ?=
+SPHINXBUILD   ?= sphinx-build
+SOURCEDIR     = source
+BUILDDIR      = build
+
+# Put it first so that "make" without argument is like "make help".
+help:
+	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+.PHONY: help Makefile
+
+# Catch-all target: route all unknown targets to Sphinx using the new
+# "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
+%: Makefile
+	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
```

Comment:
Is it worth creating a Makefile specific for sphinx when we already have a Makefile for the project?
@deusebio what do you think about this?

Reply:
I'd keep the Makefile for Sphinx. It is somewhat standard, also because one could create different format for documentation. Within the "main" Makefile, I will rather call some of these commands

Reply:
I am not sure ho to do this

Reply:
And where is the 'main' makefile?

Reply:
Do I have to launch the main Makefile for docs creation in the CI?

Reply:
Ok, I am fine with keeping this makefile separate.
@Gioia92 the Makefile will be available in the root folder as soon as PR #18 will be merged
@giovanni-prestipino since we will keep on having a Makefile also specific for Sphinx, I'd say you should use commands in this makefile to generate docs (both locally and in CI actions)